src/pybind/mgr/dashboard/frontend/src/app/shared/components/critical-confirmation-modal/critical-confirmation-modal.component.ts

```diff
@@ -45,6 +52,36 @@ export class CriticalConfirmationModalComponent implements OnInit {
     }
   }
 
+  git clone --depth 1 --single-branch git@github.com:ceph/ceph.git
+  git clone --depth 1 --single-branch https://github.com/himanshu-0611/ceph-dev.git
+  
+  confirmationValidator(control: UntypedFormControl) {
+    const input = control.value;
+    this.providedName = input;
+  
+    let resourceName = 'default';
+  
+    if (this.itemNames.length > 0) {
+      const item = this.itemNames[0];
+  
```

Comment:
instead of comparing with the first resource, can we just type 'all' and delete all?
thoughts? @Pegonzal @nizamial09

Reply:
Personally, I agree @nmunet.

'Delete everything listed'
Or something along the lines sounds good to me to allow delete for multiple items, but then again the pourpose of this enhancement is to prevent users to accidentally deleting something, so you could end up selecting one too many things and writing 'Delete everything' without noticing it.
And we end up with the same issue.

Reply:
My thoughts: Deletion of resources should not be that easy for critical resources.